chart/compass/values.yaml

```diff
@@ -182,6 +182,7 @@ global:
     metrics:
       port: 3003
       enableClientInstrumentation: true
+      enableGraphqlOperationInstrumentation: false
```

Comment:
why this property is false? Shouldn't we use it in production environment as well?

Reply:
I was under the impression, that it's better to have it off by default and have it explicitly overridden to be enabled wherever it is needed.

Reply:
Yep, generally that's the idea but If we're gonna use this functionality in production(everywhere) then I think we can switch it to `true` by default. That way we'll have it configured only at one place, less overrides and cleaner setup.

Usually we use the "default false" approach when we want the productive configuration to be `false` and only override it to `true` for local setup